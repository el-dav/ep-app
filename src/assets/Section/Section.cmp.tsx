import { styled } from 'theme';

const Section = styled.div`
  width: 100%;
  height: 100%;
  display: flex;
  flex-direction: column;
`;

export default Section;
