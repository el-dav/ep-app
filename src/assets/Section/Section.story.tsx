import React from 'react';
import SplitPane from 'react-split-pane';
import { storiesOf } from '@storybook/react';

import { ThemeProvider, styled } from 'theme';

import Section from './Section.cmp';

const Section1 = styled(Section)`
  background-color: grey;
`;
const Section2 = styled(Section)`
  background-color: coral;
`;

storiesOf('Section', module).add('Default', () => {
  return (
    <ThemeProvider>
      <SplitPane minSize={300} split="horizontal">
        <Section1 />
        <Section2 />
      </SplitPane>
    </ThemeProvider>
  );
});
