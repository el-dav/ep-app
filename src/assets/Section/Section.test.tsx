import React from 'react';
import renderer from 'react-test-renderer';

import { ThemeProvider } from 'theme';

import Section from './Section.cmp';

describe('Section', () => {
  it('renders the Component correctly', () => {
    const tree = renderer
      .create(
        <ThemeProvider>
          <Section />
        </ThemeProvider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
