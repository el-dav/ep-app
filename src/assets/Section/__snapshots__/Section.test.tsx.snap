// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section renders the Component correctly 1`] = `
<div
  className="css-1edrwtt"
/>
`;
